ts/components/services/ContactPreferencesToggles/index.tsx

```diff
@@ -69,9 +70,13 @@ const ContactPreferencesToggle: React.FC<Props> = (props: Props) => {
     [serviceId, loadServicePreference]
   );
 
+  const nav = useNavigationContext();
+
+  const isFocused = nav?.isFocused();
+
   useEffect(() => {
     loadPreferences();
```

Comment:
In this way it executes `loadPreferences` even when `isFocused` changes from `true` to `false` (when it loses the focus, I guess)?
Is it the expected behavior?

Reply:
This is a bit tricky, but when we activate through the service detail screen looks like `isFocused` doesn't updates and keeps to be `false`. In this way once we navigate back to the screen preferences informations has been updated even if the focus doesn't updates. We can check through a pair session if you prefer